introduction.md

```diff
@@ -39,8 +39,8 @@ The following diagram demonstrates the platform architecture:
 
 ## For platform builders
 
-Focused on the separation of development concerns from operational considerations through a platform-agnostic specification, the _Open Application Model_ brings modular, extensible, and portable design to building and delivering applications on platforms like Kubernetes.
+Focused on the separation of development concerns from operational considerations through a platform-agnostic specification, the _Open Application Model_ brings modular, extensible, and portable design to build and deliver applications on platforms like Kubernetes.
```

Comment:
you cannot use verbs to form the object of a sentence

Reply:
I don't think so, possibly we need an english expert to figure out this.

Reply:
This is reviewed by a MSFT technical writer. I don't know who can be called an "English expert" if she is not.....

Reply:
Sorry for bother, please forgive my poor english.